docs/current_docs/api/secrets.mdx

```diff
@@ -60,6 +60,8 @@ dagger core container \
 
 [Secret arguments](./arguments.mdx#secret-arguments) can be sourced from multiple providers: the host environment, the host filesystem, the result of host command execution, and external secret managers [1Password](https://1password.com/) and [Vault](https://www.hashicorp.com/products/vault).
 
+When a `Secret` is included in other operations, the cache for those operations will be based on the plaintext value of the secret. If the same operation is run with a secret with the same plaintext value, that operation may be cached rather than re-executed. In the above example, the secret is specified as `env://API_TOKEN`, so the cache for the `with-exec` will be based on the plaintext value of the secret. If two clients execute the container and have the same `API_TOKEN` environment variable value, they may share cache for the container. However, if two clients have different values for the `API_TOKEN` environment variable, the `with-exec` will not share cache, even though both clients specify the secret as `env://API_TOKEN`.
```

Comment:
If this section of the docs is all about secrets, shouldn't we also mention the cacheKey implicancies  here as well  on top of the cookbook? Seems something I'd expect to find here

Reply:
Yeah I wasn't sure what level of details we generally include in these sections. I can link to the cookbook from here if that's alright?

Reply:
sure, that also LGTM

Reply:
I think it's useful to include the two paragraphs that you have in the cookbook about the cache key in these pages as well for information. And with that, include a link to the cookbook for the usage example.